Add Strimzi Metrics Reporter to documentation & update CHANGELOG.md (#8)
OwenCorrigan76/strimzi-kafka-operator: documentation/modules/con-common-configuration-properties.adoc

    # ...
----

.Using Strimzi Metrics Reporter
The `metricsConfig` property contains configurations for the {StrimziMetricsReporter}.

> Strimzi supports Prometheus metrics using Strimzi Metrics Reporter to convert the metrics supported by Apache Kafka to Prometheus metrics.

Reviewer: ```suggestion
Strimzi supports Prometheus metrics using Strimzi Metrics Reporter to directly expose the metrics provided by Apache Kafka (JMX is not required).
```

To enable Strimzi Metrics Reporter, set the type to `strimziMetricsReporter`.
The `allowList` configuration is a comma separated list of regex patterns to filter the metrics that are collected. This defaults to `.*`,  which allows all metrics.

> Using `strimziMetricsReporter` directly exposes metrics in Prometheus format (no JMX dependency), and it is only supported in the Kafka component.

Reviewer: ```suggestion
NOTE: Using `strimziMetricsReporter` is only supported with the Kafka component.
```
